pkg/appgateserver/cmd/cmd.go

```diff
@@ -26,7 +26,8 @@ const omittedDefaultFlagValue = "explicitly omitting default"
 
 var (
 	flagAppGateConfig string
-	flagCosmosNodeURL string
+	flagNodeRPCURL    string
+	flagNodeGRPCURL   string
```

Comment:
I think we should add godoc comments for these as well to avoid confusing them with similar kinds of variables we use elsewhere which store the flag name (as opposed  to the value).